release/models/bgp/openconfig-bgp-common.yang

```diff
@@ -705,6 +720,10 @@ submodule openconfig-bgp-common {
       load among contributing routes proportionally to value of
       Local Administrator subfield of link-bandwidth extended
       community [draft-ietf-idr-link-bandwidth-07].
+      When set to FALSE, BGP multiplepath shall distributed traffic
```

Comment:
On the `false` case -- what are we saying here -- I think we are saying that `false` means that link bandwidth community values are ignored (sounds good), but we should clarify what is used -- are they just ECMP at that point?

Reply:
This depend on other parst of config and system itself. It may be ECMP. or it still can be wECMP, when egress interfaces speed is used to calculate weight. or weights ca be inherited form ISIS.